concepticondata/concept_set_meta/concreteness-brysbaert-2014.tsv-metadata.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
The metadata.json is for concept lists, but now you need the one we use for metadata. See here:

https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concept_set_meta/SimLex999.tsv-metadata.json

As an example.